functions/helpers.bash

```diff
@@ -42,18 +42,15 @@ cond_echo() {
 ## Add keys to apt for new package sources
 ## Valid Arguments: URL
 ##
-##    add_keys(String url)
+##    add_keys(String url, String keyFile)
 ##
 add_keys() {
-  local repoKey
-
-  repoKey="$(mktemp "${TMPDIR:-/tmp}"/openhabian.XXXXX)"
+  local repoKey="/usr/share/keyrings/${2}.gpg"
 
   echo -n "$(timestamp) [openHABian] Adding required keys to apt... "
-  cond_redirect wget -qO "$repoKey" "$1"
-  if cond_redirect apt-key add "$repoKey"; then
```

Comment:
deleted L54 but no replacement ?

Reply:
Its a one liner on line 52 now as apt-key has been deprecated in bullseye.

Reply:
Yes, it works on both. It simply changes the way apt checks the signature.